tensilelite/Tensile/LibraryIO.py

```diff
@@ -320,11 +343,10 @@ def parseLibraryLogicList(data, srcFile="?"):
                 .format(srcFile, len(data)))
 
     rv = {}
+
+    rv["LibraryLogicVersion"] = "0.0.0"
```

Comment:
We hard code the lib logic version to be 0.0.0?

Reply:
Yeah, `parseLibraryLogicList` can be removed once all the logic files are in the new format. Currently its only being used to help convert the legacy format to new format.